etc/kayobe/kolla/globals.yml

```diff
@@ -45,7 +45,7 @@ kayobe_image_tags:
     ubuntu: yoga-20230220T181235
   opensearch:
     centos: yoga-20230324T084510
-    rocky: yoga-20230324T090413
+    rocky: yoga-20230622T154317
```

Comment:
Remember to promote.

Reply:
Done: https://github.com/stackhpc/stackhpc-release-train/actions/runs/5376938075/jobs/9754763318